lerna.json

```diff
@@ -5,5 +5,6 @@
     "publish": {
       "registry": "https://registry.npmjs.org"
     }
-  }
+  },
+  "$schema": "node_modules/lerna/schemas/lerna-schema.json"
```

Comment:
This is a nitpick, but normally a "$schema" is the first property in a JSON file. This makes it more discoverable when reading the file without any context.